packages/liveblocks-core/src/lib/utils.ts

```diff
@@ -1,5 +1,8 @@
 import type { Json } from "./Json";
 
+declare const brand: unique symbol;
+export type Brand<T, TBrand extends string> = T & { [brand]: TBrand };
```

Comment:
Any reason not to use the ts-brand lib?

Reply:
My main reason is that I didn't know of its existence. Now that I do know about it, not sure if this is worth the dependency, since it's only a one-liner.